libs/wingc/src/type_check.rs

```diff
@@ -354,8 +387,9 @@ impl Subtype for Type {
 
 				// In this section, we check if the parameter types are not subtypes of each other, then this is not a subtype.
 
-				// Check that this function has at least as many required parameters as the other function requires
-				if l0.min_parameters() < r0.min_parameters() {
+				// Check that this function has at most as many required parameters as the other function requires
+				// if it doesn't, we know it's not a subtype
+				if l0.min_parameters() > r0.min_parameters() {
```

Comment:
Shouldn't this be part of how we evaluate subtypes for functions?

Reply:
That's what this code is

Reply:
Okay oops. Nice I see!

Awesome!